auth/controller/KafkaPublisher.py

```diff
@@ -46,8 +46,8 @@ def init(cls):
                     "dr_cb": True
                 },
                 "consumer": {
-                    "group.id": "dojot.auth",
-                    "metadata.broker.list": kafka_host
+                    "group_id": "dojot.auth",
```

Comment:
why it is not configurable? Other services have the option to set the group_id, isn't?
In the dojot/docker-compose#96 PR was added the environment variable KAFKA_GROUP_ID, but where it is used?

Reply:
I'll remove this part. If Kafka configuration is not set, the library will set it to its default value.